homeassistant/components/shelly/strings.json

```diff
@@ -129,6 +129,26 @@
     }
   },
   "entity": {
+    "button": {
+      "calibrate": {
+        "name": "Calibrate"
+      },
+      "close": {
+        "name": "Close"
+      },
+      "mute": {
+        "name": "Mute"
```

Comment:
I think we should change this to `Mute alarm`.

<img width="607" height="37" alt="obraz" src="https://github.com/user-attachments/assets/7feae0b0-de92-4853-b27d-c89ab163b157" />


<img width="433" height="83" alt="obraz" src="https://github.com/user-attachments/assets/2e2d5af6-f737-4921-97a2-df29eeca13ba" />

Reply:
I was trying to prevent renames in this PR, if we rename I would also rename `Self test` to `Self-test`, but in this case key name would also be needed to be renamed to be clear that it is not general mute

Reply:
Currently, one entity uses the name `Mute` (Smoke Gen1) and the other `Mute alarm` (Smoke Gen2). I think we should align this now. I also don't see a problem with changing `Self test` to `Self-test`, this request is small, so it won't be a problem to review/test it.

Reply:
Of course, if you'd like, I can prepare a PR and change these names before adding the translations. I don't want to create additional work for the translators.